dialogue-target/src/main/java/com/palantir/dialogue/Clients.java

```diff
@@ -37,7 +37,16 @@
     @Deprecated
     <T> ListenableFuture<T> call(Channel channel, Endpoint endpoint, Request request, Deserializer<T> deserializer);
 
-    EndpointChannel bind(Channel channel, Endpoint endpoint);
+    default EndpointChannel bind(Channel channel, Endpoint endpoint) {
+        return new EndpointChannel() {
+            @Override
+            public ListenableFuture<Response> execute(Request request) {
+                // this default implementation exists just in case people take new :dialogue-target but have an old
+                // version of :dialogue-serde
+                return channel.execute(endpoint, request);
+            }
+        };
```

Comment:
we probably want something like a platform long term to keep all library versions coupled.

Reply:
Yeah, so Felipe mentioned that gradle module metadata might actually be exactly what we need here